### ISSUE TITLE
[WIP] Getting ready for CKAN 2.10

### ISSUE THREAD

**Issue by amercader** [ckan/ckan#6787]
The next CKAN version will be 2.10, here's how you can help making it happen quicker:

* There are no major features under development but would be good to wrap some things up before we start the [release process](http://docs.ckan.org/en/2.9/contributing/release-process.html). I've compiled some of them in this milestone. There's a couple of Good for contribution issues in there!

    https://github.com/ckan/ckan/milestone/14

* Check how extension work against the next version. The easiest way is to enable the CI tests against the `master` branch, creating a PR that includes this change:

```diff
diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index b63fe36..fdbc771 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -17,7 +17,7 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        ckan-version: [2.9, 2.9-py2, 2.8, 2.7]
+        ckan-version: [master, 2.9, 2.9-py2, 2.8, 2.7]
       fail-fast: false
     
     name: CKAN ${{ matrix.ckan-version }}
```
This will flag test failures that will point to things to fix in the extension (or core), eg:
* [ckanext-scheming](https://github.com/ckan/ckanext-scheming/runs/5754518556?check_suite_focus=true)
* [ckanext-harvest](https://github.com/ckan/ckanext-harvest/runs/5753172278?check_suite_focus=true)

TODO

**Comment by pdelboca** [comment]
@amercader are we going to keep compatibility for `CKAN 2.7` in the extensions? 

I would suggest to only keep compatibility for `>= 2.8` since `2.7` do not run on `Python 3`.

**Comment by Zharktas** [comment]
@pdelboca Our current maintenance cycle is that 3 minor versions are supported, so 2.7 support ends once 2.10 is released. Until then it should remain in codebase and in tests.

**Comment by avdata99** [comment]
Starting check for ckanext-archiver [here](https://github.com/ckan/ckanext-archiver/pull/83)

**Comment by avdata99** [comment]
Starting check for fluent [here](https://github.com/ckan/ckanext-fluent/pull/41)

**Comment by avdata99** [comment]
Starting check `ckanext-hierarchy` [here](https://github.com/ckan/ckanext-hierarchy/pull/49)

**Comment by avdata99** [comment]
Starting check `ckanext-dcat` [here](https://github.com/ckan/ckanext-dcat/pull/208)
The workflow to run the test is awaiting approval from a maintainer

**Comment by avdata99** [comment]
We need to add `ckanext-report` considering it's a requirement for other extensions

**Comment by avdata99** [comment]
Starts with `ckanext-qa` [here](https://github.com/ckan/ckanext-qa/pull/73)

**Comment by TomeCirun** [comment]
@amercader I can work with `ckanext-googleanalytics` if no one has started yet and a polite reminder `ckanext-harvest` is ready for merging :)

**Comment by amercader** [comment]
@kojicz983 yes! As soon as we branch out the new `dev-v2.10` branch. We are waiting for a couple of big PR merges in master before doing that, but as soon as translations are ready we'll make an announcement

**Comment by pdelboca** [comment]
@amercader @smotornyuk I've started working on ckanext-googleanalytics: I would like your opinion and suggestions since the pipeline is currently broken.

https://github.com/ckan/ckanext-googleanalytics/pull/67

**Comment by amercader** [comment]
Following on #7160
